src/ui/component/copyToClipboard/copyToClipboard.tsx

```diff
@@ -0,0 +1,33 @@
+import { useAppStore } from '@/ui/store/appStore'
+import { useCallback } from 'react'
+import { Icon } from '@/ui/component/icon/icon'
+import './copyToClipboard.scss'
+
+type CopyToClipBoardProps = {
+  textToCopy: string
+  handleCopy?: (isCopied: boolean) => void
```

Comment:
Again, good catch with naming convention thanks!
Regarding the callback, I made it optional because the component still have effect because it calls `writeText`, so I would keep it optional, do you want me to change it into `mandatory` still?